rt/cmake/cmake.tdesc

```diff
@@ -0,0 +1,102 @@
+-- -*- lua -*-
+
+-- Skip the CMake test if we can't find a new enough version fo CMake
+local skip_test = true
+local cmake_path = "/bin/false"
+local cmake_ret = os.execute("cmake --version > cmake_version.txt 2>&1")
+if cmake_ret then
+   local cmake_file = io.open("cmake_version.txt")
+   local cmake_line = cmake_file:read()
+   local cmake_ver = string.match(cmake_line, "cmake version (.*)")
+   if cmake_ver >= "3.6" then
```

Comment:
does this work reliable? Better use `parseVersion`? ( see `src/parseVersion.lua`)